diff --git a/include/wcs_compat.h b/include/wcs_compat.h
index 8f07f7a..e4c3f3b 100644
--- a/include/wcs_compat.h
+++ b/include/wcs_compat.h
@@ -19,5 +19,5 @@ int wcspih(char *header, int nkeyrec, int relax, int ctrl,
            int *nreject, int *nwcs, struct wcsprm **wcs);
-int wcsset(struct wcsprm *wcs);
+int wcs_compat_wcsset(struct wcsprm *wcs);
 int wcsp2s(struct wcsprm *wcs, int ncoord, int nelem,
            const double pixcrd[], double imgcrd[], double phi[],
            double theta[], double world[], int stat[]);
diff --git a/liblt/wcs.c b/liblt/wcs.c
index 2f95136..a7b31d5 100644
--- a/liblt/wcs.c
+++ b/liblt/wcs.c
@@ -24,7 +24,7 @@ struct wcsprm *wcs_from_header(char *header, int nkey)
     if ( wcspih(header, nkey, 0, 2, &nreject, &nwcs, &wcs) )
         RAISE(E_FILE, "ERROR: reading WCS information\n")
     
-    if ( wcsset(wcs) ) 
+    if ( wcs_compat_wcsset(wcs) ) 
         RAISE(E_FILE, "ERROR: setting up wcsprm struct\n")
     else 
         return wcs;
diff --git a/liblt/wcs_compat.c b/liblt/wcs_compat.c
index 6f4c258..6c41a16 100644
--- a/liblt/wcs_compat.c
+++ b/liblt/wcs_compat.c
@@ -168,7 +168,7 @@ int wcsfree(struct wcsprm *wcs)
     return 0;
 }
 
-int wcsset(struct wcsprm *wcs)
+int wcs_compat_wcsset(struct wcsprm *wcs)
 {
     (void)wcs;
     return 0;
